packages/drivers/odsp-driver/src/getFileLink.ts

```diff
@@ -133,11 +183,16 @@ async function getFileLinkCore(
 					storageToken,
 					true,
 				);
+				// The location of file can move on Spo in which case server returns 308(Permanent Redirect) error.
+				// Adding below header will make VROOM API return 404 instead of 308 and browser can not intercept it.
+				// This error thrown by server will contain the new redirect location. Look at the 404 error parsing
+				// for further reference here: \packages\utils\odsp-doclib-utils\src\odspErrorUtils.ts
 				const requestInit = {
 					method: "POST",
 					headers: {
 						"Content-Type": "application/json;odata=verbose",
 						"Accept": "application/json;odata=verbose",
+						"prefer": "manualredirect",
```

Comment:
It is for VROOM APIs not just for Fluid scenarios. But we don't have documentation for this on their side. We knew this from our previous interactions with them when we did similar work related to redirection during load.

Reply:
It would be great to push SPO teams to document these scenarios...
@marcmasmsft - any chance you can help make it happen (or at least start the process)?